ts/features/bonus/cgn/saga/networking/bucket/index.ts

```diff
@@ -0,0 +1,12 @@
+import { delay, put } from "redux-saga/effects";
+import { cgnCodeFromBucket } from "../../../store/actions/bucket";
+
+// handle the request for CGN Otp code generation
+export function* cgnBucketConsuption(
+  // TODO Replace with the proper client once defined
+  _: any
```

Comment:
```suggestion
  _: unknown
```
I know is just temporary, but I suggest to use `unknown` to avoid `any` future problems 😛